discovery-provider/src/queries/health_check.py

```diff
@@ -92,3 +92,21 @@ def ipld_block_check():
 def ip_check():
     ip = helpers.get_ip(request)
     return success_response(ip, sign_response=False)
+
+@bp.route("/block_confirmation", methods=["GET"])
```

Comment:
Maybe this wants to live in it's own file since it's not really related to healthcheck anymore?

Reply:
this file is definitely super littered with misc. health stuff. i could go either way for now. probs makes sense to pull em all out at some point